src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/CodeMetricsAnalyzerTests.cs

```diff
@@ -559,6 +559,128 @@ End Namespace
             await VerifyBasicAsync(source, additionalText, expected);
         }
 
+        [Fact, WorkItem(2133, "https://github.com/dotnet/roslyn-analyzers/issues/2133")]
+        public async Task CA1506_CountCorrectlyGenericTypes()
```

Comment:
Can you also add this test to `CodeMetricsComputationTests.cs`?

Reply:
Done